scripts/apps/authoring/authoring/directives/AuthoringDirective.js

```diff
@@ -1026,6 +1027,9 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
                 var parts = mediaIdGenerator.getFieldParts(fieldId);
                 var field = _.find($scope.fields, (field) => field._id === parts[0]);
 
+                if (field && field.field_type === 'related_content' && relatedItemFields.indexOf(field._id) === -1) {
```

Comment:
The code you added is not related to checking if the field `isMediaField` which is what this function is for. Another reason this code should not be here is because it creates unnecessary coupling. For your code in another file(`RelationsService.ts`) to work properly, `isMediaField` has to be called exactly as it is now.

Undo changes to this file and fix it in `RelationsService.ts` instead.

Reply:
fixed it in `RelationsService.ts`